src/elements/content-sidebar/activity-feed/comment/__tests__/BaseComment.test.js

```diff
@@ -55,6 +55,10 @@ export const getWrapper = props =>
     );
 
 describe('elements/content-sidebar/ActivityFeed/comment/BaseComment', () => {
+    beforeEach(() => {
+        EditorState.moveFocusToEnd = editor => editor;
```

Comment:
Wh... wha? Just a function that returns its arg? Is this how you are clearing mocks in between tests?

Reply:
I guess I don't need to set this before every test and can just do it once. Good call.

But yeah this is how I'm mocking it. It needs to return an editor to get other tests to work. But also breaks in jsdom if we try to actually execute the real function